content/recipes/applied-examples/approach-cdisc.md

```diff
@@ -0,0 +1,167 @@
+
+
+### Assignment of properties to domains and mapping to SDTMIG variables
+
+Mapping of data dictionary properties to SDTMIG variables falls into several categories:
+
+* Direct match - a property from the data dictionary directly matches a variable in the SDTMIG 
+* Combination - two or more properties from the data dictionary need be combined into a single SDTM variable. As an example, sample collection times in the APPROACH dataset were recorded as separate variables for year, month, day, hour and minute. In SDTM, these were combined into LB.LBDTC.
+* Splitting - a property from the dataset needs to be split into component parts. This is commonly the case if values and units were captured together in the same field.
+* Derivation - some values might be derived from another value, for exampe using a logical decision tree "if A=x, then value=y" or transformed, eg converting a laboratory result reported in non-standard units into standard units.
+* Fixed value - an implicit field in the original dataset may correspond to a fixed value for an SDTMIG variable. In the APPROACH dataset, the demographics property "Age" has the unit "years" implied in the property description as the inclusion criteria for the trial specify adult subjects only. DM.AGEU is a required value so in this particular conversion, "YEARS" is set a fixed value.
+* One-to-many/many-to-one - In some cases, a single property might map to multiple SDTMIG variables, depending on the corresponding value in the dataset. Equally, multiple properties may map to the same SDTMIG variable. 
+* Unmappable properties - some project-specific properties may not have an equivalent SDTMIG variable at all. If the property and its values are important for the interpretation of the dataset overall, this scenario may require the creation of a custom domain. Equally however, some properties may simply be out of scope of SDTM. The APPROACH data dictionary contains a number of such properties, including checks whether a given CRF, examination or questionnaire was completed or sample collected, and properties to identify the coder or analyst. 
+* Auto-generated - some SDTM variables such as "STUDYID" or sequence ID ("-SEQ") will likely not be encoded directly in the dataset and need to be autogenerated or autocompleted during the data transformation process.
+
+
+### Putting together an ETL template
+
+For most of the scenarios listed above, with the exception of direct one-to-one matches, ETL scripts will need to be implemented in order to convert the data into an SDTM-compliant format. As the implementation of scripts was outside the scope of our mapping work, we instead implemented a template that could be used as a basis to generate the scripts. A copy of our template can be downloaded from [this URL](https://webdav-r3lab.uni.lu/public/datacatalog_documents//dataset/df780ac2-c79d-11ec-9b1d-acde48001122/APPROACH_CDISC-SDTM_mappings.xlsx).
+
+
```

Comment:
```suggestion
````{dropdown}
:open:
```{figure} etl-2-cdisc-figure1.png
---
name: etl-2-cdisc-figure1
alt: CDISC SDTM ETL Template 
---
CDISC SDTM ETL Template 
```
````
```

Reply:
@proccaserra I'm not entirely clear what this edit is for. Is there an existing figure you wanted to include? If not, what did you have in mind here?